src/sql/base/browser/ui/selectBox/selectBox.ts

```diff
@@ -20,6 +20,10 @@ import { SelectBoxList } from 'vs/base/browser/ui/selectBox/selectBoxCustom';
 const $ = dom.$;
 
 
+export interface SelectOptionItemSQL extends ISelectOptionItem {
+	backendValue: string; // THIS IS REQUIRED, this is the value that will actually be returned on SelectBox#values()
```

Comment:
In the HTML SELECT control the names of the displayValue vs. actualValue are "Text" and "Value".  Is value already taken for the purpose of Text in the standard control?  I would prefer `value` to `backendValue` if possible to be consistent with standard HTML.